Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu
```

Comment:
It's good to specify a version number, usually the LTS version - which would be 14.04.